### PR TITLE
Documentation revamp

### PR DESCRIPTION
**Pull request by NickIliev**
To be merged with the upcoming official release of Fiddler Everywhere

Related to https://github.com/telerik/fiddler-everywhere-docs/issues/13

### CODE REVIEW

File: user-guide/live-traffic/inspectors/inspector-types.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+- Response Header
+
+Similar to an HTTP request, every HTTP response begins with plaintext headers that describe the result from the request. The first line of the response (the _"Status line"_) contains the HTTP version (e.g. __"HTTP/1.1"__), the response status code (e.g. __"200"__), and the response status text (e.g. __"OK"__). Following the _Status line_ are one or more lines containing name-value pairs of metadata about the response and the server, such as the length of the response file, the Content-Type, and information about how the response may be cached.
+
+
+## Text
+
+The __Text__ inspector allows you to view the request and response bodies as text. The inspector truncates its display at the first null byte it finds, and as such, is unsuitable for displaying binary content.
```
Comment (ggarabedian) [inline user-guide/live-traffic/inspectors/inspector-types.md:32]: "truncates its display" sounds strange to me. Consider rephrasing to something like:
"The __Text__ inspector truncates the data it shows at the first null byte it finds, which makes it inappropriate choice for displaying binary content"

File: user-guide/live-traffic/inspectors/inspector-types.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Most of the inspector is a large text area that displays the body text interpreted using the character set detected using the headers, the byte-order-marker, or an embedded META tag declaration.
+
+![Text Inspector](../../../images/livetraffic/inspectors/inspectors-textview.png)
+
+
+## Image
+
+The __Image__ inspector (__Response only__) allows you to view the response bodies as an image. The inspector can display most common web image formats, including JPEGs, PNGs, and GIFs, as well as less common formats including cursors, WebP, JPEG-XR, bitmaps, TIFF. The inspector does not support display or SVG graphics (SVG responses can be viewed using the __Web__ inspector).
```
Comment (ggarabedian) [inline user-guide/live-traffic/inspectors/inspector-types.md:41]: 1) JPEG, PNG, GIF, without the "s" on each because you have said "formats, including..." and those are the formats.
2) Consider changing "including" in "formats including cursors, WebP" to **like**
3) The inspector does not support display or SVG graphics -> Is that correct or it should be "does not support displaying SVG graphics"?

File: user-guide/live-traffic/inspectors/inspector-types.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## WebForms
+
+The __WebForms__ inspector (__Request only__) parses the request's query string and body for any HTML form-data. If a form is found, it is parsed, and the name/value pairs are displayed in the grid view. The inspector works best with _application/x-www-form-urlencoded_ data used by most simple web forms.
+
+![WebForms Inspector](../../../images/livetraffic/inspectors/inspectors-webforms.png)
+
+
+## Web
+
+The __Web__ inspector (__Response only__) allows you to view responses in a web browser control, which provides a quick preview of how a given response may appear in a browser. The web browser control is configured to prevent additional downloads when rendering the response (to avoid flooding the __Web Sessions__ list), meaning that most images and styles will be not be displayed. Additionally, scripting and navigating are blocked, providing a read-only preview.
```
Comment (ggarabedian) [inline user-guide/live-traffic/inspectors/inspector-types.md:55]: Remove the first "be" from "be not be"

File: user-guide/live-traffic/web-sessions/web-sessions-list.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+position: 30
+---
+
+# Web Sessions List
+
+## General
+
+The Web Sessions list is an essential feature of Fiddler Everywhere - it displays a summary of each Session that Fiddler Everywhere has captured.
+
+>tip A Web Session represents a single transaction between a client and a server, sometimes known as request/response pair. Each session appears as a separate entry in the Web Sessions List (refer to the images below). Each Session object has a Request and a Response, representing what the client sent to the server and what the server returned to the client. The Session object also maintains a set of Flags that record metadata about the session, and a Timers object that stores timestamps logged in the course of processing the session.
```
Comment (ggarabedian) [inline user-guide/live-traffic/web-sessions/web-sessions-list.md:15]: This note is a repeat from the General page. Is this intended?

File: user-guide/live-traffic/web-sessions/web-sessions-list.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
+---
+title: Web Sessions List
+slug: web-sessions-list
+tags: web sessions, capturing traffic, live traffic
+publish: true
+position: 30
+---
+
+# Web Sessions List
```
Comment (ggarabedian) [inline user-guide/live-traffic/web-sessions/web-sessions-list.md:9]: The "Save", "Share" and "Remove All" sections seem to be identical to the ones in Web Sessions Toolbar (a bit more extensively explained). Consider keeping those only in one place (if needed you can link between the two articles). Also, consider having only one article on the Live Traffic if the info overlaps so much.
- You have already added a link between the articles, but more of the info still repeats and the only difference is "the button" vs "the menu".

File: user-guide/live-traffic/web-sessions/web-sessions-toolbar.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
+---
+title: Web Sessions Toolbar
+slug: web-sessions-toolbar
+tags: web sessions toolbar, fiddler toolbar, share sessions
+publish: true
+position: 20
+---
+
+# Web Sessions Toolbar
```
Comment (ggarabedian) [inline user-guide/live-traffic/web-sessions/web-sessions-toolbar.md:9]: 1) This article has a lot of images that "hide" some of the one liners of text. Consider reducing the amount of images by merging the info of 3-4 images in one and having more text in between to make it more visible.

File: user-guide/live-traffic/web-sessions/general.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+---
+title: General
+slug: web-sessions-general
+tags: web sessions, fiddler sessions, share sessions
+publish: true
+position: 10
+---
+
+# Web Sessions Section
```
Comment (ggarabedian) [inline user-guide/live-traffic/web-sessions/general.md:9]: 1) Consider using only one image with an arrow to the switch in the first image
2) Consider removing "Section" from the title
3) It feels like the **General** page does not have a clear purpose as it repeats text and images in the other two articles in this section. Consider removing it entirely.

File: user-guide/settings/privacy.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+---
+title: Privacy
+slug: privacy-submenu
+tags: privacy, delete account, unsubscribe, email list
+publish: true
+position: 40
+---
+
+# Privacy submenu
```
Comment (ggarabedian) [inline user-guide/settings/privacy.md:9]: Consider removing submenu from the title

File: user-guide/settings/https.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
+---
+title: HTTPS
+slug: decrypt-https-traffic
+tags: https, https traffic, SSL, decrypt https, https decryption, capturing HTTPS, trust certificate
+publish: true
+position: 10
+---
+
+# HTTPS Submenu
```
Comment (ggarabedian) [inline user-guide/settings/https.md:9]: Consider removing submenu from the title

File: user-guide/settings/gateway.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
+---
+title: Gateway
+slug: gateway-submenu
+tags: system proxy, manual proxy
+publish: true
+position: 30
+---
+
+# Gateway submenu
```
Comment (ggarabedian) [inline user-guide/settings/gateway.md:9]: Consider removing submenu from the title

File: user-guide/settings/connections.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
+---
+title: Connections
+slug: connections-submenu
+tags: remote connections, remote traffic, system proxy, network port
+publish: true
+position: 20
+---
+
+# Connections submenu
```
Comment (ggarabedian) [inline user-guide/settings/connections.md:9]: Consider removing submenu from the title

File: user-guide/settings/privacy.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+The __Privacy__ submenu provide options to opt-out of receiving emails from Progress and its partners and the option to delete your Fiddler Everywhere associated account.
```
Comment (ggarabedian) [inline user-guide/settings/privacy.md:11]: 1) provide**s** with "s" (or alternatively, use **contains**)
2) Rephrase "... the option to delete your ..." to avoid repeating "option" and also specifying that they can **request the deletion**, not delete it themselves (at least that's how I would expect it to work). For example:
... and the opportunity to request the deletion of your Fiddler ...

File: user-guide/settings/privacy.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+
+There are two options in the submenu:
```
Comment (ggarabedian) [inline user-guide/settings/privacy.md:13]: There are 3 items below this statement. Consider using the word "several" instead of "two" to avoid ambiguity.

File: user-guide/settings/privacy.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+
+- __Unsubscribe from emails__
+- __Delete your Fiddler Account__
+- __Submit anonymous usage statistics__ checkbox
+
+## Unsubscribe from emails
+
+To sumbit a request to unsubscribe from receiving future emails, click on the __Unsibscribe__ link. You will be redirected to a unsubscribe page where you should follow the given instructions.
+
+![Unsubscribe from emails](../../images/settings/privacy-unsub.png)
```
Comment (ggarabedian) [inline user-guide/settings/privacy.md:23]: I don't think there is a need for separate images here. You can use one without circling the items (they are pretty visible in my opinion). If you want to have them encircled, you can put a number next to the circle and mention this in the text

File: user-guide/settings/privacy.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+
+## Delete your Fiddler Account
+
+To sumbit a request to click on the __Submut "Delete Account" request__ link. You will be reditected to a page where you should follow the given instructions.
+
+![Unsubscribe from emails](../../images/settings/privacy-delete.png)
+
+## Submit anonymous usage statistics
```
Comment (ggarabedian) [inline user-guide/settings/privacy.md:31]: On the images the setting is called "send anonymous...". Unless this will be changed, you should rephrase it to match the actual UI.

File: user-guide/settings/privacy.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+
+# Privacy submenu
+
+The __Privacy__ submenu provide options to opt-out of receiving emails from Progress and its partners and the option to delete your Fiddler Everywhere associated account.
+
+There are two options in the submenu:
+
+- __Unsubscribe from emails__
+- __Delete your Fiddler Account__
+- __Submit anonymous usage statistics__ checkbox
```
Comment (ggarabedian) [inline user-guide/settings/privacy.md:17]: On the images the setting is called "send anonymous...". Unless this will be changed, you should rephrase it to match the actual UI.

File: introduction.md [hunk 17]
```diff
@@ -5,24 +5,29 @@ tags: introduction
...
 ---
+
 # Welcome to Fiddler Everywhere!
-Progress® Telerik® Fiddler is a free web debugging proxy for any browser, system or platform.
+
+**Progress® Fiddler Everywhere** is an open-source web debugging proxy for any browser, system or platform. This proxy server that runs on your Windows, Linux, or Mac platforms. Locally running programs like web browsers, MS Office applications, and other clients send their HTTP and HTTPS requests to Fiddler Everywhere, which then forwards the traffic to a web server. The server’s responses are then returned to Fiddler Everywhere, which passes the traffic back to the client. Fiddler Everywhere can decrypt HTTPS traffic and display and modify the encrypted requests using a man-in-the-middle decryption technique.  
 
 ## Licensing
+
 License agreement and detailed information about the different licenses and which one you should acquire is available in the [Licensing](https://www.telerik.com/purchase/license-agreement/fiddler) section.
```
[the remark this answers is not in the document]
Comment (NickIliev) [reply introduction.md:15]: TODO: wait for @kvelikov to push the Everywhere agreement and link it to the proper page

File: user-guide/live-traffic/inspectors/inspector-types.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+
+# Inspector Types
+
+The Inspectors allow you to view the content of each session's request and response in a variety of different formats. Fiddler supports multiple inspector types for the sessions' request (at the top) and response (at the bottom).
+
+## Headers
+
+The __Headers__ inspector, allows you to view the HTTP headers of the request and the response. By default, the content of the __Headers__ inspector is read-only and can't be edited.
+
+![Headers Inspector](../../../images/livetraffic/inspectors/inspectors-headers.png)
```
[the remark this answers is not in the document]
Comment (NickIliev) [reply user-guide/live-traffic/inspectors/inspector-types.md:17]: TODO

File: user-guide/live-traffic/web-sessions/web-sessions-list.md (hunk 5 above)
Comment (NickIliev) [reply user-guide/live-traffic/web-sessions/web-sessions-list.md:9]: We will sync on that at a later point - keeping both articles for now as the main idea was to cover the whole UI

File: user-guide/live-traffic/web-sessions/web-sessions-toolbar.md (hunk 6 above)
Comment (NickIliev) [reply user-guide/live-traffic/web-sessions/web-sessions-toolbar.md:9]: TODO

File: user-guide/live-traffic/auto-responder/rules-edtor.md [hunk 19]
```diff
@@ -0,0 +1,115 @@
...
+
+1. Click on __Create New Rule__ button in the __Rules Editor__.
+![Create New Rule button](../../../images/livetraffic/autoresponder/autoresponder-editor-create.png)
+2. Enter a __match rule__. See the [match rule section](#match-rules) for more details.
+![New Match Rule](../../../images/livetraffic/autoresponder/autoresponder-create-new-match.png)
+3. Enter an __action string__. See the [action string section](#action-strings) for more details.
+![New Action String](../../../images/livetraffic/autoresponder/autoresponder-create-new-action.png)
+4. Add the newly created rule to the __Queue Viewer__ list of rules.
+
+Tips:
```
Comment (ggarabedian) [inline user-guide/live-traffic/auto-responder/rules-edtor.md:26]: Consider making the Tips an "info/note" panel to make it more visible.

File: user-guide/live-traffic/auto-responder/rules-edtor.md [hunk 20]
```diff
@@ -0,0 +1,115 @@
...
+- __\*__ - Will match everything.
+- __EXAMPLE__ - Will match URLs of type _http://www.**EXAMPLE**.com/somePath_
+- __path1/__ - Will match URLs of type _http://www.example.com/**path1/**query=example_
+- __query__ - Will match URLs of type _http://www.example.com/path1/q=**query**_
+
+### NOT rules (for String Literals)
+
+The __NOT:__ rule is applied only if the string does not match.
+
+- __NOT:EXAMPLE__ - Will match URLs that are not containing the **EXAMPLE** string.
```
Comment (ggarabedian) [inline user-guide/live-traffic/auto-responder/rules-edtor.md:48]: For some reason I do not like "are not containing". Consider using "do not contain" instead.

File: user-guide/live-traffic/auto-responder/rules-edtor.md [hunk 21]
```diff
@@ -0,0 +1,115 @@
...
+```
+NOT:google.com
+
+www.example.com/Path1/query=foo // (MATCH)
+```
+- __NOT:path1/__ - Will match URL that are not containing **path1/**.
```
Comment (ggarabedian) [inline user-guide/live-traffic/auto-responder/rules-edtor.md:54]: Consider using "do not contain" instead.

File: user-guide/live-traffic/auto-responder/rules-edtor.md [hunk 22]
```diff
@@ -0,0 +1,115 @@
...
+```
+NOT:path2/
+
+www.example.com/path1/query=foo // (MATCH)
+www.example.com/path2/query=foo // (No Match)
+```
+
+- __NOT:query__ - Will match URLs that are not containing **query**.
```
Comment (ggarabedian) [inline user-guide/live-traffic/auto-responder/rules-edtor.md:62]: Consider using "do not contain" instead.

File: user-guide/live-traffic/auto-responder/rules-edtor.md [hunk 23]
```diff
@@ -0,0 +1,115 @@
...
+```HTML
+www.example.com/Path1/query=foo.jpg&bar // (No Match - imnproper ending)
+www.example.com/Path1/query=exam ple.jpg // (MATCH)
+www.example.com/Path1/query=foo.JPG // (No Match - mismatched case)
+www.example.com/Path1/query=somegif.gif // (MATCH)
+```
+
+## Action Strings
+
+Enter the __action string__. Beyond simply returning files, the __Auto Responder__ can perform some specific actions:
```
Comment (ggarabedian) [inline user-guide/live-traffic/auto-responder/rules-edtor.md:101]: "Enter the action string" sounds very "bloggish" :) Consider removing it.

File: user-guide/live-traffic/auto-responder/rules-edtor.md [hunk 24]
```diff
@@ -0,0 +1,115 @@
...
+
+- __filename__  - Return contents of the filename as the response.
+- __http://targetURL__ - Returns the content of the __targetURL__ as the response.
+- __*redir:http://targetURL__ - Returns a HTTP Redirect to the target URL. Unlike the simple URL rule, this ensures that the client knows where its request is going so proper cookies are sent, etc.
+- __*bpu__ - Breaks on request before hitting the server. Non-final acti0on.
```
Comment (ggarabedian) [inline user-guide/live-traffic/auto-responder/rules-edtor.md:106]: **action** instead of actio0n

File: user-guide/live-traffic/auto-responder/rules-edtor.md [hunk 25]
```diff
@@ -0,0 +1,115 @@
...
+- __*delay:####__ - Delay sending request to the server by __####__ of miliseconds. Non-final action.
+- __*header:Name=Value__ - Set the Request header with the given __Name__ to the specified __Value__. If no header of that name exists, a new header will be created. Non-final action.
+- __*flag:Name=Value__ - Set the Session Flag, with the given __Name__ to the specified __Value__. If no header of that name exists, a new header will be created. Non-final action.
+- __*CORSPreflightAllow__ - Returns a response that indicates that CORS is allowed.
+- __*reset__ - Reset the client connection immediately using a TCP/IP RST to the client.
+- __*drop__ - Close the client connection immediately without sending a response.
+- __*exit__ - Stop processing rules at this point.
+
+Rules with Non-final actions will allow the request to match multiple Auto Responder rules. As soon a rule specifying a final action is reached, the matching process exists, and no further rules are processed for that session.
```
Comment (ggarabedian) [inline user-guide/live-traffic/auto-responder/rules-edtor.md:115]: Perhaps it is a good idea to mark the "Final actions" above as well, just to make it consistent. You can also consider turning this into a table, I think it will look good - Action String | Description | Action Type
